src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimator.h

```diff
@@ -346,6 +381,18 @@ class FloatingBaseEstimator : public BipedalLocomotion::System::Advanceable<Floa
     bool setupFixedVectorParamPrivate(const std::string& param, const std::string& prefix,
                                       std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler,
                                       std::vector<double>& vec);
+
+    /**
+     * Wrapper method for getting base state from internal IMU state
+     * @param[in] state internal state of the estimator
+     * @param[in] meas previous measurement to the estimator
+     * @param[in] basePose base pose as an iDynTree Transform object
+     * @param[in] baseTwist mixe- trivialized base velocity as an iDynTree Twist object
+     */
+    bool updateBaseStateFromIMUState(const FloatingBaseEstimators::InternalState& state,
+                                     const FloatingBaseEstimators::Measurements& meas,
+                                     iDynTree::Transform& basePose,
+                                     iDynTree::Twist& baseTwist);
```

Comment:
This should be changed to `manif` and `Eigen` objects.

Reply:
I opened an issue for this. https://github.com/dic-iit/bipedal-locomotion-framework/issues/107.

This will be dealt with later in a newer PR.